extensions/amp-mowplayer/0.1/amp-mowplayer.js

```diff
@@ -255,20 +254,13 @@ class AmpMowplayer extends AMP.BaseElement {
   }
 
   /**
-   * Sends 'listening' message to the Mowplayer iframe to listen for events.
+   * Check player is visible or not based on breakpoint and send message to player
    * @private
    */
-  listenToFrame_() {
-    if (!this.iframe_) {
-      return;
-    }
-
-    this.sendCommand_('listening', [
-      'amp',
-      window.location.href,
-      window.location.origin,
-      true,
-    ]);
+  onVisibilityObserver_() {
+    const {intersectionRatio} = this.element.getIntersectionChangeEntry();
+    const visible = intersectionRatio > 0.5 ? true : false;
+    this.sendMessage_('visibility_observer_visibility', {'visible': visible});
```

Comment:
`getIntersectionChangeEntry` is gone, you need [`measureIntersection`](https://github.com/ampproject/amphtml/blob/1b3e642edf999142688baa233c27e854c9f9dfac/src/utils/intersection.js#L71)

```suggestion
    measureIntersection(this.element).then((entry) => {
      const {intersectionRatio} = entry;
      const visible = intersectionRatio > 0.5 ? true : false;
      this.sendMessage_('visibility_observer_visibility', {'visible': visible});
    });
```